core/src/main/scala/akka/kafka/CommitterSettings.scala

```diff
@@ -7,11 +7,51 @@ package akka.kafka
 import java.util.concurrent.TimeUnit
 
 import akka.actor.ActorSystem
+import akka.annotation.ApiMayChange
 import akka.util.JavaDurationConverters._
 import com.typesafe.config.Config
 
 import scala.concurrent.duration._
 
+@ApiMayChange(issue = "https://github.com/akka/alpakka-kafka/issues/882")
+sealed trait CommitDelivery
+
+/**
+ * Selects how the stream delivers commits to the internal actor.
+ */
+@ApiMayChange(issue = "https://github.com/akka/alpakka-kafka/issues/882")
+object CommitDelivery {
+
+  /**
+   * Expect replies for commits, and backpressure the stream if replies do not
+   * arrive.
+   */
+  case object Ask extends CommitDelivery
```

Comment:
Looks good.  I don't want to get hung up on the naming, but tell/ask does rely on the users knowledge of Akka message delivery to understand.  The scaladoc comments are descriptive though so I think it's good as-is.

It may be worth noting that there are several Kafka configurations that use the terminology `acks`.  In the producer client properties it's used for acknowledgements of message receipt across all replicas.  In the broker there's a setting `offsets.commit.required.acks`.  TBH I'm not really sure what it's used for.

Reply:
I'm open to naming suggestions. I believe `ack` interferes with how things are called in Kafka. 
Maybe `ExpectResponses` and `SendAndForget` are better? But that's is incorporated in the Akka Actor inspired names, of course.

Reply:
We agreed on `WaitForAck` and `SendAndForget`.